docs/detections/rules-ui-create.asciidoc

```diff
@@ -276,6 +349,27 @@ image::images/rule-actions.png[]
 * Continue with <<rule-notifications>>.
 * Create the rule (with or without activation).
 
+[discrete]
+[[preview-rules]]
+=== Preview your rule (optional)
+
+You can preview a custom query, threshold, or EQL (Event Correlation) rule to get feedback on how noisy the rule will be before submitting it. This allows you to fine-tune the query, if needed, to reduce the amount of alerts that may come in.
+
+To preview a rule:
+
+. Write the rule query.
++
+. Select a timeframe of data to preview query results -- `Last hour` or `Last day` -- from the *Quick query preview* drop-down.
++
+. Click *Preview results*. A histogram shows the number of events you can expect based on the defined rule parameters and historical events in your indices.
```

Comment:
The histogram shows the number of "alerts" a user can expect. I think `number of events` in this case might be too general.